node/execution_test.go

```diff
@@ -116,94 +115,3 @@ func finalizeExecution(t *testing.T, executor coreexecutor.Executor, ctx context
 	err := executor.SetFinal(ctx, 1)
 	require.NoError(t, err)
 }
-
-func TestExecutionWithDASync(t *testing.T) {
-	t.Run("basic DA sync with transactions", func(t *testing.T) {
-		require := require.New(t)
-
```

Comment:
Questionable what this is testing tbh, it just produces a block which is already tested in full_node_integration_test.go